app/scripts/modules/core/src/pipeline/filter/ExecutionFilters.tsx

```diff
@@ -151,7 +150,7 @@ export class ExecutionFilters extends React.Component<IExecutionFiltersProps, IE
   };
 
   private updatePipelines(pipelines: IPipeline[]): void {
```

Comment:
you could return the promise from this fn

Reply:
:+1: done!